scrapper.py

```diff
@@ -35,7 +46,31 @@ def find_articles(self):
         """
         Finds articles
         """
-        pass
+        headers = {
+            'user-agent': "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36"
+        }
+
+        sleep_period = 5
+        for url in self.seed_urls:
+            response = requests.get(url, headers=headers)
+            time.sleep(sleep_period)
+
+            if not response.ok:
+                print("Request failed")
```

Comment:
it is a weak check. probably we do not want to process such an URL then? what do you think?

Reply:
if response is no okay, we should continue